lib/application/Backbone.js

```diff
@@ -1,6 +1,11 @@
 "use strict";
 
 var Backbone = require("backbone").noConflict();
+var Environment = require("../environment/Environment");
+
+if (Environment.clientDebuggingEnabled() && window.__backboneAgent) {
+    window.__backboneAgent.handleBackbone(Backbone);
```

Comment:
the implementation is described here https://github.com/Maluen/Backbone-Debugger#backbone-detection